extensions/smallrye-jwt/deployment/src/main/java/io/quarkus/smallrye/jwt/deployment/SmallRyeJwtBuildTimeConfig.java

```diff
@@ -22,4 +22,10 @@ public interface SmallRyeJwtBuildTimeConfig {
      */
     @WithDefault("SunRsaSign")
     String rsaSigProvider();
+
+    /**
+     * If it should generate or not public and private keys for generating JWT tokens.
+     */
+    @WithDefault("true")
+    boolean generateSignKeys();
```

Comment:
The reason users want to have properties set automatically if for them to avoid setting properties :-), the fact that no verification/signing keys are set is an indication that the keys have to be generated :-), so you can drop this property